src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -110,13 +109,12 @@ export const StakingOpportunities = ({
   const handleClick = useCallback(
     (values: Row<UserStakingOpportunityWithMetadata>) => {
       const { chainId, assetReference, assetNamespace } = fromAssetId(assetId)
-      const provider = chainIdToLabel(chainId)
       const { account: validatorAddress } = fromAccountId(values.original.id)
       const [opportunityAccountId] = deserializeUserStakingId(values.original.userStakingId)
       history.push({
         search: qs.stringify({
           accountId: routeAccountId ?? opportunityAccountId,
-          provider,
+          provider: DefiProvider.Cosmos,
```

Comment:
1. The `DefiProvider` for Cosmos SDK staking should be `DefiProvider.Cosmos`. `DefiProvider.Osmosis` is used for LP only, so this was wrong
2. `<StakingOpportunities />` goes away entirely in https://github.com/shapeshift/web/pull/3841 either way